elements/rh-footer/rh-footer-social-link.css

```diff
@@ -1,10 +1,15 @@
 :host {
   display: block;
 
+  /* @Deprecated with PFE 2.0 */
   --pfe-icon--color: var(--_icon-color);
```

Comment:
we can drop it in this pr then

Reply:
should it be 

```css
pfe-icon {
  color: var(--_icon-color);
}
```
?

Reply:
> we can drop it in this pr then

It will be a breaking change to existing sites if we remove it.  I was thinking about marking it has deprecated for now.

Reply:
> should it be
> 
> ```css
> pfe-icon {
>   color: var(--_icon-color);
> }
> ```

`pfe-icon` is in the lightdom of `rh-footer-social-link`.  We did this to keep its parent `<a>` in the lightdom.